### PR TITLE
fix(docs): updated the quickstart for kubeflow pipelines. Fixes 4617 in (kubeflow/pipelines)

### PR DESCRIPTION
**Pull request by NikeNano**
Relates to: https://github.com/kubeflow/pipelines/issues/4617 updated with new images and update the names as well. Moved the deploy instructions to refer to the in depth instructions instead and suggested standalone as the recommended for testing.

### CODE REVIEW

File: content/en/docs/pipelines/pipelines-quickstart.md [hunk 1]
```diff
@@ -23,53 +18,25 @@ Kubeflow Pipelines.
...
-      documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-
-    1. Run ```kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80``` and go to `http://localhost:8080/`.
-
-1. Click **Pipelines** to access the pipelines UI. The pipelines UI looks like
-  this:
-  <img src="/docs/images/pipelines-ui.png" 
-    alt="Pipelines UI"
-    class="mt-3 mb-3 border border-info rounded">
+Once you have deployed kubeflow pipelines make sure you can access the UI, this step depends on the method of deployment.
```
Comment (JohanWork) [inline content/en/docs/pipelines/pipelines-quickstart.md:23]:
```suggestion
Once you have deployed kubeflow pipelines make sure you can access the UI, how to do it varied depending on the deployment method. 
```

File: content/en/docs/pipelines/pipelines-quickstart.md [hunk 2]
```diff
@@ -92,65 +59,27 @@ workload:
     alt="Run results on the pipelines UI"
     class="mt-3 mb-3 border border-info rounded">
 
-You can find [the source code for the basic parallel join sample](https://github.com/kubeflow/pipelines/blob/master/samples/core/parallel_join/parallel_join.py)
+You can find [the source code for the data passing in python components sample](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
```
Comment (JohanWork) [inline content/en/docs/pipelines/pipelines-quickstart.md:62]:
```suggestion
You can find [the source code for the python components](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components)
```

**Comment by Bobgy** [comment]
Thank you so much! This looks great!

/LGTM
/Assign @joeliedtke 
In case you want to review

**Comment by Bobgy** [comment]
Note, the rendered page can be viewed at https://deploy-preview-2512--competent-brattain-de2d6d.netlify.app/docs/components/pipelines/pipelines-quickstart/

File: content/en/docs/components/pipelines/pipelines-quickstart.md [hunk 3]
```diff
@@ -2,74 +2,38 @@
...
-      documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
-
-    1. Run ```kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80``` and go to `http://localhost:8080/`.
-
-1. Click **Pipelines** to access the pipelines UI. The pipelines UI looks like
-  this:
-  <img src="/docs/images/pipelines-ui.png" 
-    alt="Pipelines UI"
-    class="mt-3 mb-3 border border-info rounded">
+Once you have deployed Kubeflow Pipelines make sure you can access the UI, this step depends on the method of deployment.
```
Comment (joeliedtke) [inline content/en/docs/components/pipelines/pipelines-quickstart.md:20]: Suggested revision (grammar, clarity): 

Once you have deployed Kubeflow Pipelines, make sure you can access the UI. The steps to access the UI vary based on the method you used to deploy Kubeflow Pipelines.

File: content/en/docs/components/pipelines/pipelines-quickstart.md [hunk 4]
```diff
@@ -2,74 +2,38 @@
...
-    {{% pipelines-compatibility %}} 
+## Deploy Kubeflow and open the Kubeflow Pipelines UI
 
-1. When Kubeflow is running, access the Kubeflow UI at a URL of the form
-  `https://<deployment-name>.endpoints.<project>.cloud.goog/`, as described in the setup
-  guide. The Kubeflow UI looks like this:
-  <img src="/docs/images/central-ui.png" 
-    alt="Kubeflow UI"
-    class="mt-3 mb-3 border border-info rounded">
+There are several options to [deploy Kubeflow Pipelines](/docs/components/pipelines/installation/), follow the option that best suits your needs. If you are uncertain and just want to try out kubeflow pipelines it is recommended to start with the [standalone deployment](/docs/components/pipelines/installation/standalone-deployment/).
```
Comment (joeliedtke) [inline content/en/docs/components/pipelines/pipelines-quickstart.md:18]: Nit: I would recommend linking directly to the installation options page

[deploy Kubeflow Pipelines](/docs/components/pipelines/installation/overview/)

File: content/en/docs/components/pipelines/pipelines-quickstart.md [hunk 5]
```diff
@@ -2,74 +2,38 @@
...
-  <img src="/docs/images/pipelines-ui.png" 
-    alt="Pipelines UI"
-    class="mt-3 mb-3 border border-info rounded">
+Once you have deployed Kubeflow Pipelines make sure you can access the UI, this step depends on the method of deployment. 
 
 ## Run a basic pipeline
 
-The pipelines UI offers a few samples that you can use to try out
-pipelines quickly. The steps below show you how to run a basic sample that
+Kubeflow Pipelines offers a few default samples that you can use to try out
```
Comment (joeliedtke) [inline content/en/docs/components/pipelines/pipelines-quickstart.md:24]: Remove "default"

File: content/en/docs/components/pipelines/pipelines-quickstart.md [hunk 6]
```diff
@@ -92,65 +56,27 @@ workload:
     alt="Run results on the pipelines UI"
     class="mt-3 mb-3 border border-info rounded">
 
-You can find [the source code for the basic parallel join sample](https://github.com/kubeflow/pipelines/blob/master/samples/core/parallel_join/parallel_join.py)
-in the Kubeflow Pipelines repo.
+You can find the source code for the **Data passing in python components** tutorial
```
Comment (joeliedtke) [inline content/en/docs/components/pipelines/pipelines-quickstart.md:59]: It's better to use link text that describes the content that we are linking to. Suggested revision

You can find the [source code for the **Data passing in python components** tutorial](https://github.com/kubeflow/pipelines/tree/master/samples/tutorials/Data%20passing%20in%20python%20components) in the Kubeflow Pipelines repo.

File: content/en/docs/components/pipelines/pipelines-quickstart.md [hunk 7]
```diff
@@ -159,30 +85,20 @@ Follow these steps to set up the necessary GCP services and run the sample:
...
 1. Explore the graph and other aspects of your run by clicking on the 
   components of the graph and the other UI elements. The following screenshot
-  shows the graph when the pipeline has finished running:
+  shows part of the graph when the pipeline has finished running:
     <img src="/docs/images/pipelines-xgboost-graph.png" 
       alt="XGBoost results on the pipelines UI"
       class="mt-3 mb-3 border border-info rounded">
 
-You can find the source code for the XGBoost training sample in the 
+You can find the source code for the **XGBoost - Iterative model training** demo in the
```
Comment (joeliedtke) [inline content/en/docs/components/pipelines/pipelines-quickstart.md:93]: Suggested revision:

You can find the [source code for the **XGBoost - Iterative model training** demo](https://github.com/kubeflow/pipelines/tree/master/samples/core/xgboost_training_cm) in the 
Kubeflow Pipelines repo.

**Review by joeliedtke — CHANGES_REQUESTED** [review]
Thanks for updating this doc, I've added a few suggestions.

**Comment by NikeNano** [comment]
> Thanks for updating this doc, I've added a few suggestions.

Thanks for the feedback, update with your suggestions @joeliedtke

**Comment by joeliedtke** [comment]
/lgtm
/approve

Thanks for updating this page, @NikeNano!

**Pull request merged**